packages/odyssey-stylelint/src/index.js

```diff
@@ -36,6 +37,14 @@ module.exports = {
       "always",
       { ignoreAtRules: ["if", "else"] },
     ],
+    "declaration-empty-line-before": [
+      "always",
+      {
+        except: ["after-declaration"],
+        ignore: ["after-comment", "first-nested", "inside-single-line-block"],
+        severity: "warning",
+      },
+    ],
```

Comment:
why the rule changes here? Seems strange that this would not be accounted for in the stylelint-prettier config preset?

Reply:
I took another look at this with fresh eyes, I don't think it's necessary and has been removed. Thanks for pointing it out!